lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1240,6 +1283,8 @@ def test_basic(self, mock_emit):
             "endorsed_comment_list_url": None,
             "non_endorsed_comment_list_url": None,
             "editable_fields": ["abuse_flagged", "following", "raw_body", "title", "topic_id", "type", "voted"],
+            'read': False,
+            'has_endorsed': False
```

Comment:
1) Why is it `'has_endorsed'` in some places and `'endorsed'` in others?
2) Why `endorsed` instead of `answered`?